build.zig

```diff
@@ -3,6 +3,8 @@ const build_facilio = @import("facil.io/build.zig").build_facilio;
 
 pub fn build(b: *std.build.Builder) !void {
     const target = b.standardTargetOptions(.{});
+    if (target.getOsTag() == .windows)
+        @panic("Facil (and Zap) doesn't support Windows, at least not currently. Try targeting Linux and using WSL\n");
```

Comment:
Please rephrase like: "Currently, Facil.io and Zap are not compatible with Windows. Consider using Linux or Windows Subsystem for Linux (WSL) instead."